File: backend/src/main/scala/kpn/api/common/ChangeSetSubsetElementRefs.scala
package kpn.api.common

import kpn.api.custom.Subset

case class ChangeSetSubsetElementRefs(
  subset: Subset,
  elementRefs: ChangeSetElementRefs
) {

  def elementIds: Seq[Long] = elementRefs.elementIds

  def happy: Boolean = elementRefs.happy

  def investigate: Boolean = elementRefs.investigate
}

File: backend/api/src/main/java/kpn/api/common/ChangeSetSubsetElementRefs.java
package kpn.api.common;

import kpn.api.common.ChangeSetElementRefs;
import kpn.api.custom.Subset;

public record ChangeSetSubsetElementRefs(
  Subset subset,
  ChangeSetElementRefs elementRefs
) {}

/* TODO migrate

  def elementIds: Seq[Long] = elementRefs.elementIds

  def happy: Boolean = elementRefs.happy

  def investigate: Boolean = elementRefs.investigate

*/
